src/jdk.hotspot.agent/macosx/native/libsaproc/libproc_impl.h

```diff
@@ -86,39 +86,39 @@ typedef struct reg {
 #elif defined(aarch64)
 /*** registers copied from bsd/arm64 */
 typedef struct reg {
-   register_t     r_r0;
```

Comment:
I would verge on whitespace change noise, but the realignment makes sense. Up to you.